### ISSUE TITLE
JEA - Errors when attempting to Start-Service in a JEA session

### ISSUE THREAD

**Issue by doctordns** [PowerShell/PowerShell#14650]
I have a JEA session setup. I attach the session configuration and role capabilities files.

Here are the files
[JEA issue.zip](https://github.com/PowerShell/PowerShell/files/5855630/JEA.issue.zip)

This configuration (attempts to) enable ```Start-Service``` and ```Stop Service``` cmdlets to allow starting/stopping the DNS Service.

I see two specific issues:
1. If I run a script block to ```{Start-Service -Name DNS}```, I get an error that the ```Name``` parameter is unknown. If I attempt to pipe the service name (eg ```{'DNS" | Start-Service}``` I get no error, but the service does not start. I checked this on other services as well.
2. Using ```Get-PSSessionCapability``` also does not correctly show the ```Start-Service``` or ```Stop-Service``` cmdlets as cmdlets. They are shown as functions. This makes sense for the DNS Server cmdlets as they are loaded via WPS Compat, but the cmdlets are, well cmdlets not functions.

## Steps to reproduce

1. Define JEA endpoints per the attached
2. Run Get-PSSessionCapability 

## Expected behavior
1. Enabling start-service to start the DNS service
2. Showing Start-Service as a cmdlet not a function

## Actual behavior

1. For first issue:
```console
PS C:\Foo> $sb4 = {Start-Service -Name DNS -Verbose}
Invoke-Command -ScriptBlock $SB4 @ICMHT
Start-Service: A parameter cannot be found that matches parameter name 'Name'.
```

2. For second issue:
```console
PS C:\Foo> # 11. Viewing remoting endpoints
Get-PSSessionConfiguration  |
  Format-Table -Property NAME, PSVersion, Run*Account

Name             PSVersion RunAsVirtualAccount
----             --------- -------------------
DnsAdminsJEA     7.1       True
PowerShell.7     7.1       false
PowerShell.7.1.1 7.1       false

PS C:\Foo> # 12. Verifying what the user can do
$SCHT = @{
  ConfigurationName = 'DnsAdminsJEA'
  Username          = 'Reskit\JerryG'
}
Get-PSSessionCapability  @SCHT |
  Sort-Object -Property Module

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Alias           clear -> Clear-Host
Function        Stop-Service
Function        Start-Service
Function        Select-Object
Function        Restart-Computer
Function        Out-Default
Function        Measure-Object
Function        Get-HW
Function        Get-Help
Application     ipconfig.exe                                       10.0.2028… C:\WINDOWS\system32\ipconfig.exe
Function        Get-Command
Function        Exit-PSSession
Function        Clear-Host
Function        Get-FormatData
Application     whoami.exe                                         10.0.2028… C:\WINDOWS\system32\whoami.exe
Alias           gh -> Get-Help
Alias           gcm -> Get-Command
Alias           select -> Select-Object
Alias           cls -> Clear-Host
Alias           exsn -> Exit-PSSession
Alias           measure -> Measure-Object
Function        Set-DnsServerClientSubnet                          2.0.0.0    DnsServer
```

## Environment data

Name                           Value
----                           -----
PSVersion                      7.1.1
PSEdition                      Core
GitCommitId                    7.1.1
OS                             Microsoft Windows 10.0.20285
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, 5.0, 5.1.10032.0, 6.0.0, 6.1.0, 6.2.0, 7.0.0, 7.1.1}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```none

```

**Comment by doctordns** [comment]
Purely, as an aside, how much testing is done with JEA? Are there tests performed prior to shipping a new version? If so, pointers please?  If not, I am unclear how to add them, but the attached files are a good starting point (IMHO!).

**Comment by TravisEz13** [comment]
remoting working group:  This needs more investigation.  https://github.com/PowerShell/PowerShell/files/5855630/JEA.issue.zip

**Comment by K0nne** [comment]
Hello,

we are also trying to use powershell functions in a JEA session via VisibleFunctions and get similar errors:

`Select-Object: A parameter cannot be found that matches parameter name 'ExpandProperty'.`

Before "select-object" was added to the VisibleFunctions section of the role capability file. 
Since then we also got the following warning message:

`
WARNING: The 'Microsoft.PowerShell.Utility' module was not imported because the 'Microsoft.PowerShell.Utility' snap-in was already imported.`

The warning led me to the following JEA issue. 
It was pointed out that this might be an powershell issue. Also a possible solution was given:
https://github.com/PowerShell/JEA/issues/42#issuecomment-373504502

I hope this is related.

Our system (server 2016):
```
Name                           Value
----                           -----
PSVersion                      5.1.14393.3866
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.3866
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
